ocs_ci/ocs/ui/views.py

```diff
@@ -453,6 +454,80 @@
         By.XPATH,
     ),
     "cluster-set-selection": ("//a[normalize-space()='{}']", By.XPATH),
+    "cc_create_cluster": ("createCluster", By.ID),
+    "cc_provider_vmware_vsphere": ("//*[@id='vmware-vsphere']", By.XPATH),
+    "cc_cluster_name": ("input[placeholder='Enter cluster name']", By.CSS_SELECTOR),
```

Comment:
How will this xpath utilize the cluster name?

Reply:
changed to xpath.